queryserver-it/src/it/java/org/apache/phoenix/end2end/SecureQueryServerPhoenixDBIT.java

```diff
@@ -324,6 +324,18 @@ public static synchronized void stopKdc() throws Exception {
 
     @Test
     public void testBasicReadWrite() throws Exception {
+        File file = new File(".");
+        runShellScript("python", Paths.get(file.getAbsolutePath(), "src", "it", "bin", "test_phoenixdb.py").toString());
```

Comment:
If it's easier, we could also bind some action to `pre-integration-test` and `post-integration-test` which start a PQS instance. I remember trying to debug failing python tests were a major pain in the rear with the I/O redirection flowing back through the Java maven process.